ui/src/app/store/node/types.ts

```diff
@@ -0,0 +1,82 @@
+import type { Model } from "app/store/types";
```

Comment:
Same again, `node` isn't a slice, so I think these types should probably live somewhere else. Perhaps these should live in the `store/types.ts` module?

Reply:
I'm thinking maybe store/types should be a dir with store/types/node.ts etc. to make it a bit more obvious where those types live. I'm sure we'll have a lot of shared types that will need to live externally to avoid circular imports.

Reply:
store/types/node.ts